ForkClone.md

```diff
@@ -39,14 +39,16 @@
     Clone this repo to your local machine using
     ```git clone git@github.com:gbowne1/codebooker.git```.
 
-    Once you have successfully cloned the repository, a new sub-directory will appear in the location where you ran the clone repository commands. This new sub-directory will have the same name as the original repository that was cloned.
+    After you have successfully cloned the repo, a new sub-directory appears in the location where you ran the clone repo commands. This new sub-directory has the same name as the original repo that you cloned.
```

Comment:
I changed "once" to "after" for precise language and changed the tense to present tense.